<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>552</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>0</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Simple SDL2 Mixer X Music player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../_resources/musicplayer.qrc">
    <normaloff>:/cat_musplay/cat_musplay_48x48.png</normaloff>:/cat_musplay/cat_musplay_48x48.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);
}
QPushButton {
	background-color: rgb(65, 65, 65);
	border-color: rgb(80, 80, 80);
	border-style: outset;
    border-width: 2px;
	color: rgb(0, 255, 255);
	border-radius: 3px;
    height: 25px;
 }
QPushButton:pressed {
    background-color: rgb(80, 80, 80);
    border-style: inset;
}
QCheckBox {
	border-color: rgb(0, 170, 0);
    border-width: 0px;
	color: rgb(0, 255, 255);
 }
QComboBox, QSpinBox{
	border-radius: 3px;
    border-width: 2px;
    border-style: solid;
	border-color: rgb(0, 170, 0);
	color: rgb(0, 255, 255);
}
QGroupBox
{
    padding: 5px;
    border-width: 1px;
    border-style: solid;
    border-color: rgb(0, 170, 0);
	border-radius: 3px;
}
QLabel
{
	background-color: transparent;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="adlmidi_xtra">
         <property name="title">
          <string>ADLMIDI Extra Setup</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>ADL Midi bank ID:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="tremolo">
            <property name="text">
             <string>Tremolo</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="vibrato">
            <property name="text">
             <string>Vibrato</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QCheckBox" name="modulation">
            <property name="text">
             <string>Scalable modulation</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="fmbank">
            <property name="currentIndex">
             <number>-1</number>
            </property>
            <property name="maxVisibleItems">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QCheckBox" name="adlibMode">
            <property name="text">
             <string>Ad-Lib drums mode</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="QPushButton" name="resetDefaultADLMIDI">
            <property name="text">
             <string>Reset to defaults</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Volume ranges model:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="volumeModel">
            <item>
             <property name="text">
              <string>[Auto]</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Generic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CMF</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>DMX</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Apogee</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9X</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QCheckBox" name="logVolumes">
            <property name="text">
             <string>Use logarithmic volumes (Generic and CMF volume models only)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="gme_setup">
         <property name="title">
          <string>Game Music Emulators setup</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Track number:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="trackID">
            <property name="maximum">
             <number>9999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="midi_setup">
         <property name="title">
          <string>MIDI Setup</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>MIDI Device:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="mididevice">
            <item>
             <property name="text">
              <string>ADL Midi (OPL Synth emulation)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Timidity (GUS-like wavetable MIDI Synth)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Native midi (Built-in MIDI of your OS)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FluidSynth (SF2-based wavetable MIDI Synth)</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>45</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="open">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+O</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="play">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
         <property name="shortcut">
          <string>Space</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="shortcut">
          <string>.</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volume">
         <property name="maximum">
          <number>128</number>
         </property>
         <property name="value">
          <number>128</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="smallInfo">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QCheckBox" name="recordWav">
      <property name="text">
       <string>Record WAV</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="musArtist">
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgba(0, 0, 0, 0);
border-bottom-color: rgb(0, 120, 0);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="musAlbum">
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgba(0, 0, 0, 0);
border-bottom-color: rgb(0, 120, 0);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Press &quot;Open&quot; or drag&amp;drop music file into this window</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="musTitle">
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgba(0, 0, 0, 0);
border-bottom-color: rgb(0, 120, 0);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="musCopyright">
      <property name="styleSheet">
       <string notr="true">border-width: 1px;
border-style: solid;
border-color: rgba(0, 0, 0, 0);
border-bottom-color: rgb(0, 120, 0);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>trackID</tabstop>
  <tabstop>mididevice</tabstop>
  <tabstop>fmbank</tabstop>
  <tabstop>tremolo</tabstop>
  <tabstop>vibrato</tabstop>
  <tabstop>modulation</tabstop>
 </tabstops>
 <resources>
  <include location="../_resources/musicplayer.qrc"/>
 </resources>
 <connections/>
</ui>
